readme.md

```diff
@@ -132,21 +132,24 @@ AVA comes with an intelligent watch mode. [Learn more in its recipe](docs/recipe
 ```console
 $ ava --help
 
+  Futuristic test runner 🚀
+
```

Comment:
This is intentionally left out as it just duplicates our readme and repo description.